actionList.js

```diff
@@ -6426,6 +6426,8 @@ Action.Escape = new Action("Escape", {
         unlockTown(7);
     },
     story(completed) {
+        //FIXME: This will (unfortunately) give the story completion for creating the looping potion, even
+        //if the player didn't, because completing story N will also complete all stories less than N.
```

Comment:
Yeah, that needs some bigger refactoring than I feel like doing right at this moment and I've been putting it off XD